### PR TITLE
Added support for twitter email

### PR DESCRIPTION
**Pull request by r3lax**
Twitter now allows email access if the twitter app is configured to view it.

### CODE REVIEW

**Comment by markbates** [comment]
@r3lax the tests appear to be failing.

**Comment by r3lax** [comment]
The test is failing because twitter_test.go asserts the email address being empty.  This PR provides email data.

File: providers/twitter/twitter_test.go [hunk 1]
```diff
@@ -114,6 +114,7 @@ func mockTwitter(f func(*httptest.Server)) {
 			"profile_image_url": "http://example.com/image.jpg",
 			"id_str":            "1234",
 			"location":          "Springfield",
+			"email":			 "duffman@springfield.com",
```
Comment (markbates) [inline providers/twitter/twitter_test.go]: Can you please run Go fmt on this code? Thanks.

**Comment by markbates** [comment]
Thanks.

**Pull request merged**